roles/common/tasks/serverspec.yml

```diff
@@ -1,8 +1,12 @@
 ---
+- name: clean all serverspec definition
```

Comment:
Yes. since we need to clean all other roles' existing specs, and I don't want to add removal code in every roles.
It will be replaced by Inspec in next release.